devops/cloudbuild-pr.yaml

```diff
@@ -16,10 +16,12 @@ steps:
   - name: node:16.14.2
     entrypoint: npm
     env:
+      - 'VUE_APP_ADDRESS_COMPLETE_KEY=$VUE_APP_ADDRESS_COMPLETE_KEY'
       - 'VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID=$_VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID'
       - 'VUE_APP_GENESYS_ENV=$_VUE_APP_GENESYS_ENV'
       - 'VUE_APP_GENESYS_URL=$_VUE_APP_GENESYS_URL'
       - 'VUE_APP_GENESYS_ID=$_VUE_APP_GENESYS_ID'
+      - 'VUE_APP_REGISTRIES_SEARCH_API_KEY=$VUE_APP_REGISTRIES_SEARCH_API_KEY'
```

Comment:
These both have a typo. Should be `$_xxx`.

Reply:
Sorry about that. My brain is overheating 🤯

Reply:
\<boom\>

No, that was me.